Mathlib/NumberTheory/FLT/Three.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2024 Riccardo Brasca. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Riccardo Brasca
+-/
+import Mathlib.Data.ZMod.Basic
+
+/-!
+# Fermat Last Theorem in the case `n = 3`
+The goal of this file is to prove Fermats Last theorem in the case `n = 3`.
```

Comment:
```suggestion
The goal of this file is to prove Fermat Last theorem in the case `n = 3`.
```
If we are still making suggestions to this PR... I'm happy with a different choice for Fermat's Last Theorem, but I prefer consistency, over the exact form.